apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -46,17 +47,24 @@ export class AITagView extends React.Component {
   }
 
   validateImage = async () => {
+    const FILE_LIMIT = 5000000; // 5MB limit from AWS Rekognition
+    const FILE_DIMENSION_LIMIT = 80; // 80px limit
     const imageId = get(this.props.entries.image.getValue(), 'sys.id');
-    if (!imageId) { return true; }
+    if (!imageId) { return ; }
```

Comment:
Should this function return a bool? I know `undefined` is falsey but it seems more semantic to make sure the return value is something expected.

```suggestion
    if (!imageId) { return false; }
```

In any case, do we not use prettier in this app?
```suggestion
    if (!imageId) { return; }
```

Reply:
It's a `void` fn. I didn't want to return a bool which is what I did before but it ended up confusing me why I was returning a value in the first place.

Reply:
Gotcha, thanks for the explanation.